apps/editing-toolkit/editing-toolkit-plugin/wpcom-coming-soon/styles.scss

```diff
@@ -0,0 +1,166 @@
+@import '~@automattic/typography/styles/fonts';
+
+html {
+	/* No admin bar nor marketing bar on this page */
+	margin-top: 0 !important;
+}
+.body {
+	background: #117ac9;
+	color: #fff;
+	display: grid;
+	font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen-Sans', 'Ubuntu',
+		'Cantarell', 'Helvetica Neue', sans-serif;
+	grid-gap: 24px;
+	-ms-grid-columns: ( 1fr ) [1];
+	grid-template-columns: repeat( 1, 1fr );
+	padding-right: 24px;
+	padding-left: 24px;
+}
+.inner {
+	align-items: flex-end;
+	display: flex;
+	flex-wrap: wrap;
+	-ms-grid-column: 1;
+	grid-column-start: 1;
+	-ms-grid-column-span: 1;
+	grid-column-end: span 1;
+	height: 100vh;
+	justify-content: space-between;
+}
+.main,
+.marketing {
+	flex: 0 0 100%;
+}
+.name {
+	color: #fff;
+	font-size: 19px;
+	line-height: 1.3;
+	margin-bottom: 8px;
+	padding: 0;
+	text-align: left;
+}
+.description {
+	color: #fff;
+	font-size: 40px;
+	line-height: 1.15;
+	padding: 0;
+	text-align: left;
+}
+.description,
+.copy {
+	font-family: $serif;
+}
+.marketing {
+	padding-bottom: 8px;
+}
+.marketing-copy {
+	display: flex;
+	align-items: center;
+}
+.logo {
+	height: 32px;
+	margin-right: 16px;
+	width: 32px;
+}
+.copy {
+	line-height: 1.4;
+	margin: 0;
+}
+.marketing-buttons .button {
+	background: #fff;
+	border-radius: 2px;
+	border: 1px solid #fff;
+	box-sizing: border-box;
+	color: #117ac9;
+	display: block;
+	font-size: 16px;
+	font-weight: 700;
+	line-height: 21px;
+	padding: 13px;
+	text-align: center;
+	text-overflow: ellipsis;
+	text-decoration: none;
+	transition: opacity 0.15s ease-out;
+	white-space: nowrap;
+	width: 100%;
+}
+.marketing-buttons .button-secondary,
+.marketing-buttons .button-secondary:hover,
+.marketing-buttons .button-secondary:focus {
+	background: transparent;
+	color: #fff;
+}
+.marketing-buttons .button:hover,
+.marketing-buttons .button:focus {
+	opacity: 0.85;
+}
+@media screen and ( min-width: 660px ) {
+	.description,
+	.copy {
+		font-family: $brand-serif;
```

Comment:
This works 👍 

`font-weight` would be good to add.

Alternatively you can do `@extend .wp-brand-font;` or add `wp-brand-font` selector directly in HTML.

Reply:
Trying not to diverge from the source too far just yet, someone probably needs to iterate on this as a whole. The linter is complaining about font sizes at the moment too.

Reply:
👍 Yup, just suggestion, definitely not a blocker for shipping. 

You probably have good reasons for custom font sizes since this isn't Calypso UI per-se, so you could just add linter ignore rule on top of the file?